test/topology_raft_disabled/test_raft_upgrade_basic.py

```diff
@@ -53,3 +50,8 @@ async def test_raft_upgrade_basic(manager: ManagerClient, random_tables: RandomT
     assert(rs)
     logging.info(f"group0_history entry description: '{rs[0].description}'")
     assert(table.full_name in rs[0].description)
+
+    logging.info("Booting new node")
+    await manager.server_add(config={
+        'consistent_cluster_management': True
```

Comment:
At least a comment is missing that we are actually overriding the pre-existent setting to False. consistent_cluster_management: True is set by default in the current master, so it is not obvious why we have to add the very same setting here.

Reply:
The suite is called `topology_raft_disabled`. The whole **point** of the test is to have Raft disabled by default, after all it's an upgrade test. Adding a comment like "the default is False" is completely redundant. I can add it but there's no point really.